save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/OrganizationController.kt

```diff
@@ -114,6 +114,23 @@ internal class OrganizationController(
         return Mono.just(response)
     }
 
+    /**
+     * @param organizationName
+     * @param authentication
+     */
+    @DeleteMapping("/{organizationName}/delete")
+    @PreAuthorize("isAuthenticated()")
+    fun deleteOrganization(@PathVariable organizationName: String, authentication: Authentication): Mono<StringResponse> {
+        val role = lnkUserOrganizationService.getGlobalRoleOrOrganizationRole(authentication, organizationName)
+        val response = if (role.priority >= Role.OWNER.priority) {
```

Comment:
Can we use `organizationPermissionEvaluator` here, like you in ProjectController?

Reply:
ok, will do it